eng/pipelines/jobs/pack-sign-publish.yml

```diff
@@ -42,3 +42,15 @@ jobs:
         /p:DotNetPublishUsingPipelines=true
         /p:ThirdPartyNoticesFilePath='$(Build.SourcesDirectory)/$(_TPNFile)'
       displayName: Pack, Sign, and Publish
+    - task: powershell@2
+      displayName: Generate Build Info
+      inputs:
+        targetType: inline
+        script: |
+            New-Item "$(Build.ArtifactStagingDirectory)/buildInfo" -Type Directory
+            Copy-Item "$(Build.SourcesDirectory)/artifacts/packages/Release/Shipping/*.buildversion" "$(Build.ArtifactStagingDirectory)/buildInfo/buildversion"
```

Comment:
Btw, there are multiple build version files; now, they should all have the same value _except_ if we start versioning some package versions differently (e.g. diagnostics repo does this for the NETCore.Client library where it has a version of 0.2.0). Maybe this should copy the files as-is without rename and then let the consume pick which file it wants to key off of. If we need to construct build version files with just the package name, I'm okay with that. It's likely we'll do away with the existing *.buildversion and *.version files once this change is in place.